package.json

```diff
@@ -57,6 +57,7 @@
     "babel-plugin-import": "^1.11.0",
     "babel-polyfill": "^6.26.0",
     "lodash": "^4.17.15",
+    "lodash.debounce": "^4.0.8",
```

Comment:
because api-explorer sub package only installs specific lodash modules

https://github.com/mia-platform/api-explorer/blob/master/packages/api-explorer/package.json

If I don't add this line I get a warning inside the sub package that lodash.debounce is not listed in the project deps

Reply:
This is not the package-json of the modules, I don't expect where I use this library I should install lodash.debounce

Reply:
ok I've fixed it